### ISSUE TITLE
WordPress 5.5 removes jQuery migrate

### ISSUE THREAD

**Issue by cjhaas** [bu-ist/bu-navigation#52]
WordPress 5.5 is no longer enqueuing [jQuery Migrate](https://core.trac.wordpress.org/changeset/48323) by default so methods like [`live()`](https://api.jquery.com/live/) (which was removed in 1.9) no longer work out of the box. Because of this, we're starting to see console errors such as:

```
deletion.min.js Uncaught TypeError: a(...).live is not a function
    at HTMLDocument.<anonymous> (deletion.min.js?ver=1.2.6:1)
    at i (load-scripts.php?c=0&load[chunk_0]=jquery,utils&ver=ae81108254863c9270fda7a0f121f9e8:2)
    at Object.fireWith [as resolveWith] (load-scripts.php?c=0&load[chunk_0]=jquery,utils&ver=ae81108254863c9270fda7a0f121f9e8:2)
    at Function.ready (load-scripts.php?c=0&load[chunk_0]=jquery,utils&ver=ae81108254863c9270fda7a0f121f9e8:2)
    at HTMLDocument.J (load-scripts.php?c=0&load[chunk_0]=jquery,utils&ver=ae81108254863c9270fda7a0f121f9e8:2)
```

And these are bleeding into core feature, specifically the link dialog with errors such as:
```
wplink.min.js?ver=ae81108254863c9270fda7a0f121f9e8:2 Uncaught TypeError: Cannot read property 'show' of undefined
    at Object.open (wplink.min.js?ver=ae81108254863c9270fda7a0f121f9e8:2)
    at acf.Model.open (acf-input.min.js?ver=5.8.13:2)
    at n.onClickEdit (acf-input.min.js?ver=5.8.13:2)
    at n.<anonymous> (acf-input.min.js?ver=5.8.13:1)
    at HTMLAnchorElement.d (load-scripts.php?c=0&load[chunk_0]=jquery,utils&ver=ae81108254863c9270fda7a0f121f9e8:2)
    at HTMLDivElement.dispatch (load-scripts.php?c=0&load[chunk_0]=jquery,utils&ver=ae81108254863c9270fda7a0f121f9e8:3)
    at HTMLDivElement.r.handle (load-scripts.php?c=0&load[chunk_0]=jquery,utils&ver=ae81108254863c9270fda7a0f121f9e8:3)
```

**Comment by surkum** [comment]
The severity of this bug is high as the js loading process crash which leads to other plugins js code not to be loaded and executed.

**Comment by jdub233** [comment]
Thanks for reporting this, we are looking into it!

**Comment by jdub233** [comment]
We are testing the fix from @cjhaas and are planning a new release with #55 for this update.

**Comment by nobleclem** [comment]
Is this going to get released via the wordpress plugins site?

**Comment by jdub233** [comment]
Yes, we will publish this update to the wordpress.org plugin site.

**Comment by jdub233** [comment]
Should be fixed with #55

This latest version has also been released on wordpress.org:
https://wordpress.org/plugins/bu-navigation/
